service/src/main/kotlin/fi/espoo/evaka/reports/StartingPlacementsReport.kt

```diff
@@ -66,6 +67,15 @@ private fun Database.Read.getStartingPlacementsRows(year: Int, month: Int): List
 
     return createQuery(sql)
         .bind("range", FiniteDateRange.ofMonth(year, Month.of(month)))
-        .mapTo<StartingPlacementsRow>()
-        .toList()
+        .map { rs, _ ->
```

Comment:
By the way, if you use the single parameter `.map` interface you can use our `mapColumn` helper and you won't have to manually pick types or anything. Eg.
```
.map { mapper -> mapper.mapColumn("child_id") }
```

Reply:
Didn't know about that, thank you!